charts/hlf-k8s/values.yaml

```diff
@@ -300,6 +300,12 @@ enrollments:
   csrHost: service-hostname
 
 hooks:
+  # Service account used for the hooks, must not be part of the release and have the rights to delete secrets
+  serviceAccount:
+    name: ""
+    # Note that this will set the namespace of the deleteSecret hook as serviceAccounts are namespaced resources
+    # If empty the namespace will be set to the release namespace during render
```

Comment:
I think we should remove the comments here, because the same comments are already present in the readme and the two sets of comments will go out of sync sooner or later. Having the comments in a single, up-to-date place seems preferable.